MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,491 @@
+---
+layout: post
+title: Getting Started with Maui DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion Maui DataForm (SfDataForm) control, its elements and more.
+platform: Maui
+control: SfDataForm
+documentation: ug
+---
+
+**SfDataForm**
+
+The `Maui DataForm` (SfDataForm) control helps editing the data fields of any data object. It can be used to develop various forms such as login, reservation, data entry, etc. Key features includes the following:
+
+* Layout and grouping: Supports to linear, grid layout and floating label layout with grouping support. Supports customizing the layout with different heights for each item.
+* Caption customization: Supports loading the image as caption for the editor.
+* Editors: Built-in support for text, numeric, numeric up-down, picker, date picker, time picker, switch,drop-down,autoComplete and checkbox editors. 
```

Comment:
Editors: Built-in support for text, multiline text, password, picker, date picker, time picker, switch, ComboBox, AutoComplete, RadioGroup and checkbox editors.